src/bin/inx-chronicle/cli.rs

```diff
@@ -15,35 +15,39 @@ pub struct ClArgs {
     /// The location of the configuration file.
     #[arg(short, long, env = "CONFIG_PATH")]
     pub config: Option<String>,
-    /// The MongoDB connection string.
-    #[arg(long, env = "MONGODB_CONN_STR")]
-    pub mongodb_conn_str: Option<String>,
     /// The url pointing to an InfluxDb instance.
-    #[arg(long, env = "INFLUXDB_URL")]
+    #[arg(long = "influxdb.url", env = "INFLUXDB_URL")]
```

Comment:
Nope, don't like this.

Reply:
you mean the dot notation?

Reply:
Yeah but they probably have a reason for that, I.E. they parse the dot notation to set variables directly in the config or something. We don't.

Reply:
Changed